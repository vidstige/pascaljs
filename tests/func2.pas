program Func2;
function Add(A, B: Integer): Integer;
begin
    Add := A + B;
end;
function IsEven(i: Integer): Integer;
begin
    IsEven := (i mod 2) = 0;
end;
begin
  WriteLn('Is 3 + 4 even? ', IsEven(Add(3, 4)));
end.