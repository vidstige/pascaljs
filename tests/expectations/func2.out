Is 3 + 4 even? false
